### DISCUSSION TITLE
Is the tool usable with other editors?

### DISCUSSION THREAD

**Discussion by mkoncek** [redhat-developer/vscode-java#3146] [Q&A] [answered]
Is this server able to communicate via standard input / output?
If so, can you provide instructions to build it?
It would be great if it could be used with other editors such as Kate.

**Comment by fbricon** [comment, marked as answer]
The underlying Java Language server (Eclipse JDT.LS) is used in other editors. See https://github.com/eclipse/eclipse.jdt.ls#running-from-the-command-line for some instructions. You can also look at how other [clients](https://github.com/eclipse/eclipse.jdt.ls#clients) run it.

**Reply by mkoncek** [reply]
Thanks, I thought this project implemented its own server.
